spec/features/users/create_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+RSpec.feature 'User', type: :feature do
+  scenario 'Creates a new User' do
+    visit 'http://localhost:3000/users/sign_up'
+
+    within find('#new_user') do
+      fill_in 'user_name', with: 'RSpec'
+      fill_in 'user_email', with: 'rspec@something.com'
+      fill_in 'user_password', with: '123456'
+      fill_in 'user_password_confirmation', with: '123456'
+      click_button 'commit'
+    end
+  rescue StandardError => e
+    puts e.message.to_s
+  end
+end
+
+RSpec.describe User, type: :model do
+  describe '#name' do
+    before :each do
+      User.create(name: 'test', email: 'test@test.com')
+    end
```

Comment:
Please use the `let` syntax instead of `before :each` This is a practice we've been encouraging since the previous section! :smiley: